adapt-and-extend/theming/resourceregistry.rst

```diff
@@ -18,47 +18,75 @@ Introduction to Plone 5 resources
 
 Plone 5 introduces new concepts for working with JavaScript and CSS in Plone.
 
-We finally have a solution to define JavaScript modules and their dependencies - in contrast to the previous approach, where all dependencies had to be loaded in order.
-We achieve that with the tool RequireJS, which follows the Asynchronous Module Definition (AMD) approach.
-We chose AMD over other module loading implementations (like CommonJS) because AMD can be used in non-compiled form in the browser and also compiled Through-The-Web.
-This way we can also support a "development mode" like in Plone versions prior to 5, where changes in the JavaScript sources are instantly reflected (actually they are made available a few seconds after initial browser load).
+JavaScript and AMD
+------------------
+
```

Comment:
@cewing We should state somewhere that we can write non-AMD JS in an add-on and registrering it in a bundle and that's 100% valid.
AMD is a good practice, and it is provided by default in Plone 5, so that's really easy to use in our context, but it is not mandatory.
A bundle containing something like:
```
$(document).ready(function() {console.log("Hey, I am not AMD!!!");})
 ```
will work fine.
I think it is important to tell it explicitly so people who are afraid of frontend feel more confortable.